src/main/java/org/wickedsource/docxstamper/el/DefaultEvaluationContextConfigurer.java

```diff
@@ -0,0 +1,32 @@
+package org.wickedsource.docxstamper.el;
+
+import org.springframework.expression.TypeLocator;
+import org.springframework.expression.spel.SpelEvaluationException;
+import org.springframework.expression.spel.SpelMessage;
+import org.springframework.expression.spel.support.*;
+import org.wickedsource.docxstamper.api.EvaluationContextConfigurer;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * {@link EvaluationContextConfigurer} that has a better default security,
+ * especially doesn't allow Especially known injections.
+ */
+public class DefaultEvaluationContextConfigurer implements EvaluationContextConfigurer {
+    @Override
+    public void configureEvaluationContext(StandardEvaluationContext context) {
+        TypeLocator typeLocator = typeName -> {
+            throw new SpelEvaluationException(SpelMessage.TYPE_NOT_FOUND, typeName);
+        };
+        context.setPropertyAccessors(List.of(DataBindingPropertyAccessor.forReadWriteAccess()));
+        context.setConstructorResolvers(Collections.emptyList());
+        context.setMethodResolvers(new ArrayList<>(List.of(DataBindingMethodResolver.forInstanceMethodInvocation())));
+        context.setBeanResolver(null);
+        context.setTypeLocator(typeLocator);
+        context.setTypeConverter(new StandardTypeConverter());
+        context.setTypeComparator(new StandardTypeComparator());
+        context.setOperatorOverloader(new StandardOperatorOverloader());
+    }
```

Comment:
Hey thanks for taking my suggestion 😄 

If a breaking change is okay it would be a lot easier to allow setting the evaluation context from outside. It should default to org.springframework.expression.spel.support.SimpleEvaluationContext, to restore default behaviour the users can set the context to org.springframework.expression.spel.support.StandardEvaluationContext

If breaking the signatures is not okay, we could deprecate configureEvaluationContext but still allow it, and calling it will also set the EvaludationContext to StandardEvaluationContext

Reply:
Yes, thanks a lot. I will not break the signature right now, as I know some user won't be able to handle it quickly.

In the meantime, I'll tighten the vuln, before having a safer default (in the form of the simple standard execution you proposed) in the next major version I plan to release around November.